python/lsst/ts/criopy/m1m3/ForceActuator/GraphPageWidget.py

```diff
@@ -31,7 +32,7 @@ class GraphPageWidget(Widget):
     callbacks to trigger updates.
     """
 
-    def __init__(self, m1m3: MetaSAL):
+    def __init__(self, m1m3: MetaSAL | Simulator):
```

Comment:
You could create an interface type that defines what `m1m3` should implement and use it instead of the "or". Check "typing Protocol" for some ideas. I am not really sure what is in `MetaSAL` and `Simulator` but you could see what common interface is used there.

Reply:
I have similar thoughts, but let's keep that simple. Can address this in the future. The interface problem is it might be too complex, particularly given MetaSAL is a Python ABC.